code/modules/mob/living/silicon/robot/component.dm

```diff
@@ -95,6 +96,7 @@
 	external_type = /obj/item/robot_parts/robot_component/binary_communication_device
 	energy_consumption = 0
 	max_damage = 30
+	component_disabled = 0
```

Comment:
not needed, you've defined component_disabled above, this will inherit the defined properties.